sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/api/Azure.ResourceManager.RecoveryServices.net8.0.cs

```diff
@@ -240,6 +278,26 @@ public static partial class ArmRecoveryServicesModelFactory
         public static bool operator !=(Azure.ResourceManager.RecoveryServices.Models.BackupStorageVersion left, Azure.ResourceManager.RecoveryServices.Models.BackupStorageVersion right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct BCDRSecurityLevel : System.IEquatable<Azure.ResourceManager.RecoveryServices.Models.BCDRSecurityLevel>
```

Comment:
this should be `BcdrSecurityLevel`

Reply:
BTW please search `Bcdr` in this file thoroughly to ensure all the ocurrence of `bcdr` has the same casing.